dist/ghp_updater.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+ABSOLUTE_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "${ABSOLUTE_PATH}/commons.sh"
+DOC_OUTPUT_DIR="${PROJECT_PARENT_DIR}/target/doc"
+GH_REPO="shiywang/cincinnati"
```

Comment:
We should respect `CINCINNATI_GITHUB_PUSH_TOKEN_PATH` which could exist in CI and point to a file rather than containing the token directly.

```suggestion
GH_REPO="shiywang/cincinnati"
CINCINNATI_GITHUB_PUSH_TOKEN="${CINCINNATI_GITHUB_PUSH_TOKEN:-$(cat ${CINCINNATI_GITHUB_PUSH_TOKEN_PATH})"
```